CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## 0.9.0-dev
   **NEW FEATURES**
+  -[#221] (https://github.com/Datatamer/unify-client-python/issues/221) delete an attribute mapping
```

Comment:
I think there needs to be a space between the - and [ and not between ] and (
(check how the markdown renders on github)